docker-app/qfieldcloud/core/tests/test_api.py

```diff
@@ -26,3 +41,42 @@ def test_api_status_cache(self):
         toc = time.perf_counter()
 
         self.assertGreater(toc - tic, 0)
+
+    def test_api_default_paginator_offset(self):
```

Comment:
Is it possible change your test to aligni with the already existing way of writing tests like here and the rest of the API testing code:

https://github.com/opengisch/qfieldcloud/blob/d7941b3614f161a08f8cbb6f03599496e2857c46/docker-app/qfieldcloud/core/tests/test_user.py#L101-L113

Or there is a technical limitation for that?

Reply:
I think this is the case now? Else I think the difference is necessary, because I don't need to play around with tokens.

Reply:
Using now this style (client-based) style of authentication on the second test added in this PR. The first test still uses a factory because I need to be able to get a Request object to pass to the ProjectViewSet.as_view() closure. The reason for this is that it's the only way to ensure that the response with correspond to the actual rendered response (and not just the serialized response).